dv_utils/connectors/file.py

```diff
@@ -25,6 +27,32 @@ def __init__(self, config: FileConfiguration) -> None:
             self.config.file_name = urlparse(self.config.url).netloc
 
     def get(self):
-        response = requests.get(self.config.url)
-        with open(os.path.join(self.config.download_directory, self.config.file_name), 'w') as file:
-            file.write(response.text)
+        if(self.__should_use_scraper()):
+            scraper = cloudscraper.create_scraper()
+            response = scraper.get(self.config.url)
+        else:
+            response = requests.get(self.config.url)
+
+        is_valid = self.__handle_response(response)
+
+        if(is_valid):
+            with open(os.path.join(self.config.download_directory, self.config.file_name), 'w') as file:
+                file.write(response.text)
+
+    def __should_use_scraper(self) -> bool:
+        use_scraper_cleaned = self.config.use_scraper.strip().lower()
+        if(use_scraper_cleaned == "true"):
+            return True
+        
+        if(use_scraper_cleaned == "false"):
+            return False
```

Comment:
plaats je deze logica naar de populate_configuration? Dan zijn booleans direct supported. 
Ook misschien iets als `use_scraper_cleaned in ['true', '1', <andere dingen misschien nog>]`